### DISCUSSION TITLE
Adjust shut down timer?

### DISCUSSION THREAD

**Discussion by myfriedrice** [hass-agent/HASS.Agent#129] [Q&A] [answered]
Is it possible to adjust the timer on the shutdown function?
![image](https://github.com/user-attachments/assets/40ff80de-57b8-4a2a-99a3-2715b79e9653)
Or ignore timer and just shutdown immediately?

**Comment by amadeo-alex** [comment, marked as answer]
Hello, the one minute delay is the default one used after executing "shutdown".
If you'd like to have custom delay or no delay at all you can use the "Custom Command" with "shutdown" and appropriate parameters - https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/shutdown

For example to shutdown immediately you can use:
```
shutdown /s /f /t 0
```
